modular_skyrat/master_files/code/modules/client/preferences/mutant_parts.dm

```diff
@@ -0,0 +1,1292 @@
+/datum/preference/toggle/allow_mismatched_parts
```

Comment:
If there was a way to make it so all the preferences about mismatched parts were just appearing underneath the existing options, that'd be neat, just to reduce a bit of the cognitive load on the users.